Milvus.Client/MilvusCollection.Entity.cs

```diff
@@ -554,7 +564,7 @@ public Task<FlushResult> FlushAsync(CancellationToken cancellationToken = defaul
                     Value = Math.Min(batchSize, leftItemsCount).ToString(CultureInfo.InvariantCulture)
                 });
 
-            var nextExpression = pkField.DataType == DataType.VarChar
+            string nextExpression = pkField.DataType == DataType.VarChar
```

Comment:
In the concatenation below (`nextExpression += $" and {userExpression}"`), aren't you missing parentheses around the user expression, to prevent any possible operator precedence issues?

Reply:
added